src/components/externalCosts/rangeChart.js

```diff
@@ -193,10 +192,12 @@ const RangeChart = ({ data, currency, height, aggregateBy }) => {
   });
 
   return (
-    <ResponsiveContainer height={height} width={"100%"}>
       <BarChart
         data={orderedBars}
         margin={{ top: 30, right: 35, left: 30, bottom: 45 }}
+      responsive
+      height={height}
+      width="100%"
```

Comment:
Total nit: can these match indentation with the other component props?

Reply:
Note to self: just create a pre-commit hook that runs Prettier against the codease.